Serverless/Azure Functions/About Azure Functions.md

```diff
@@ -1,36 +1,57 @@
-Azure Functions is a Function-as-a-Service ("FaaS"), i.e. a platform for running configurable blocks of code that run in response to some event. Functions support several different programming languages such as C# and JavaScript, F#, Java, Python, and PowerShell.
+[Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview) is Microsoft's serverless solution. They are, unsurprisingly, hosted on Azure, Microsoft's cloud service. Functions support several different programming languages such as C# and JavaScript, F#, Java, Python, and PowerShell.
 
-The main benefits of using Azure Functions and serverless solutions are:
-
-* Writing less code - Allows the developer to focus on the business logic implementation and utilize the ready-to-use infrastructure features.
-* Maintain less infrastructure - Cloud infrastructure providers manage servers, security, and scaling. Functions are scaled automatically.
-* Save on costs - No need to buy resources in advance - pay as you go.
+Azure Functions are interlocked with Azure Storage services, meaning that every Azure function needs Azure storage configured to be able to run. The storage account connection is used by the Functions host for operations such as managing triggers and logging function executions. It's also used when dynamically scaling function apps.
```

Comment:
I get the idea, but I personally don't like the `interlocked` word, could we put some word instead that is more "user-friendly"? 😄

Reply:
Agreed, I rewrote the entire sentence, now it's more reader-friendly and it even contains more info 😄